docs/UserGuide.md

```diff
@@ -183,10 +199,10 @@ _Details coming soon ..._
 
 Action | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
+**Add** | `add q/DESCRIPTION a/DESCRIPTION​` <br> e.g., `add q/hi a/hola`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+**Edit** | `edit CARD_NUMBER q/DESCRIPTION a/DESCRIPTION`<br> e.g.,`edit 1 q/hi a/hola`
+**Flip** | `flip CARD_NUMBER`<br> e.g., `flip 3`
+**Test** | `test CARD_NUMBER a/CORRECT_ANSWER`<br> e.g., `test 1 a/good morning`
+**Upload** | `upload CSV_FILE_PATH`<br> e.g., `upload ./dictionary.csv`
+**Help** | `help [COMMAND]`<br> e.g., `help, help add`
```

Comment:
Same comment as above, perhaps it would be better if we let each person update based on their assigned command so that the work done for each issue is clearly shown?

Reply:
`add` has already been updated and merged into master